components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/OldTokensCleanDAO.java

```diff
@@ -84,7 +102,15 @@ public void cleanupTokenByTokenValue(String token, Connection connection) throws
     }
 
     private void saveTokenInAuditTable(OldAccessTokenDO oldAccessTokenDAO, Connection connection) throws SQLException {
-        PreparedStatement insertintoaudittable = connection.prepareStatement(SQLQueries.STORE_OLD_TOKEN_IN_AUDIT);
+
+        String sql;
+        if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
+            sql = SQLQueries.STORE_OLD_TOKEN_IN_AUDIT_WITH_IDP_NAME;
+        } else {
+            sql = SQLQueries.STORE_OLD_TOKEN_IN_AUDIT;
+        }
+
+        PreparedStatement insertintoaudittable = connection.prepareStatement(sql);
```

Comment:
please change the variable 'insertintoaudittable' to camel case.

Reply:
There are few more code to be refactored related to old token cleanup. I have created this issue to track it https://github.com/wso2/product-is/issues/5195